.env

```diff
@@ -1,6 +1,5 @@
 # A web3 endpoint provider
-# WEB3_PROVIDER="https://mainnet.infura.io/v3/..."
-WEB3_PROVIDER="https://web3.dappnode.net"
+WEB3_PROVIDERS=https://eth.llamarpc.com,https://eth-goerli.api.onfinality.io/public
```

Comment:
Umm, maybe this should not go here? I don't think we have to deliver the code with a pre-configured and private web3 endpoint

Reply:
I think that is a good practice to deliver the code with a template of configuration, we can replace those public endpoints with dummy urls